sdk/eventgrid/azure-messaging-eventgrid/src/samples/java/com/azure/messaging/eventgrid/samples/EventGridNamespaceSample.java

```diff
@@ -0,0 +1,85 @@
+package com.azure.messaging.eventgrid.samples;
+
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.models.CloudEvent;
+import com.azure.core.models.CloudEventDataFormat;
+import com.azure.core.util.BinaryData;
+import com.azure.messaging.eventgrid.EventGridClient;
+import com.azure.messaging.eventgrid.EventGridClientBuilder;
+import com.azure.messaging.eventgrid.models.AcknowledgeOptions;
+import com.azure.messaging.eventgrid.models.ReceiveDetails;
+import com.azure.messaging.eventgrid.models.ReceiveResult;
+import com.azure.messaging.eventgrid.models.RejectOptions;
+import com.azure.messaging.eventgrid.models.ReleaseOptions;
+import com.azure.messaging.eventgrid.samples.models.User;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class EventGridNamespaceSample {
```

Comment:
`Namespace` is term we haven't used in any of the clients or docs. So, we should just drop that term here.